lib/players/human.rb

```diff
@@ -0,0 +1,8 @@
+class Player::Human < Player
+
+  def move(board)
+    puts "Please enter your move from 1 - 9"
+    input = gets.chomp
+  end
```

Comment:
You don't need to store your return value in a local variable; just utilize the implicit return value of the expression.
```
def move(board)
  puts "Please enter your move from 1 - 9"
  gets.chomp
end
```